src/amo/components/Addon/index.js

```diff
@@ -154,22 +151,18 @@ export class AddonBase extends React.Component {
     this.props.toggleThemePreview(event.currentTarget);
   }
 
-  headerImage({ compatible }) {
```

Comment:
`headerImage` is still being called at line 435 with `{ compatible: isCompatible }`.